source/globalCommands.py

```diff
@@ -1710,6 +1710,22 @@ def script_activateInputGesturesDialog(self, gesture):
 	script_activateInputGesturesDialog.__doc__ = _("Shows the NVDA input gestures dialog")
 	script_activateInputGesturesDialog.category=SCRCAT_CONFIG
 
+	def script_reportActiveConfigurationProfile(self, gesture):
+		activeProfileName = config.conf.profiles[-1].name
+
+		if not activeProfileName:
+			# Translators: Message announced when the command to report the current configuration profile and
+			# the default configuration profile is active.
+			activeProfileMessage = _("normal configuration profile active")
+		else:
+			# Translators: Message announced when the command to report the current configuration profile
+			# is active. The placeholder '{profilename}' is replaced with the name of the current active profile.
+			activeProfileMessage = _("{profileName} configuration profile active").format(profileName=activeProfileName)
+		ui.message(activeProfileMessage)
+	# Translators: Input help mode message for the report current configuration profile command.
+	script_reportActiveConfigurationProfile.__doc__ = _("Reports the name of the current NVDA configuration profile")

```

Comment:
Have you considered using script decorator here?

Reply:
I have pushed a new commit to use the decorator now.